gotocompany/optimus/integration/v1beta1/event.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package gotocompany.optimus.core.v1beta1;
+
+import "google/protobuf/timestamp.proto";
+import "gotocompany/optimus/core/v1beta1/resource.proto";
+import "gotocompany/optimus/core/v1beta1/job_spec.proto";
+
+option go_package = "github.com/goto/proton/optimus";
+option java_multiple_files = true;
+option java_outer_classname = "Event";
+option java_package = "com.gotocompany.proton.optimus";
+
+message ResourceChangePayload {
+  string datastore_name = 1;
+  ResourceSpecification resource = 2;
+}
+
+message JobChangePayload {
+  string job_name = 1;
+  JobSpecification jobSpec = 2;
+}
+
+message JobRunPayload {
```

Comment:
- Can we also publish the job start time here? 
- Is there a concept of retry of jobs in Optimus? If so, will it be possible to publish the attempt number for this job run (in case there were some failures previously)?

Reply:
Sure, we can send both the information. They will be added as part of separate pr.

Reply:
Will address this in later pr.